From d5fc639ef91cdcac2837aad99e934fe51866648e Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 21 Mar 2017 15:59:44 +0800
Subject: [PATCH] printf-parse.c: Fix build on Visual Studio 2008

We didn't have a stock stdint.h to include and so intmax_t is not
defined.  Add a define for it for Visual Studio 2008.
---
 gettext-runtime/intl/printf-parse.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gettext-runtime/intl/printf-parse.c b/gettext-runtime/intl/printf-parse.c
index ebcb484..fdbcce0 100644
--- a/gettext-runtime/intl/printf-parse.c
+++ b/gettext-runtime/intl/printf-parse.c
@@ -56,6 +56,9 @@
 # if HAVE_INTTYPES_H_WITH_UINTMAX
 #  include <inttypes.h>
 # endif
+# if (_MSC_VER < 1600)
+#  define intmax_t long long
+# endif
 #else
 # include <stdint.h>
 #endif
-- 
2.10.2.windows.1

